app/forms/mappings/Constraints.scala

```diff
@@ -86,6 +86,22 @@ trait Constraints {
         Invalid(errorKey, maximum)
     }
 
+  protected def required(errorKey: String): Constraint[Option[String]] =
+    Constraint {
+      case Some(str) => {
+        val trimmedStr = str.replace("\n", " ")
+          .replace("\r", " ")
+          .replaceAll(" +", " ")
+          .trim
+        if (trimmedStr.nonEmpty) {
+          Valid
+        } else {
+          Invalid(errorKey)
+        }
+      }
+      case None => Invalid(errorKey)
+    }
```

Comment:
I don't think this is needed, because there already should be a `text` mapping that deals with it being mandatory/required field.